src/FakeItEasy.Analyzer/UnusedReturnValueAnalyzer.cs

```diff
@@ -42,13 +39,8 @@ private static void AnalyzeCall(SyntaxNodeAnalysisContext context)
                 return;
             }
 
-            var diagnosticName = GetDiagnosticName(methodSymbol);
-            if (diagnosticName == null)
-            {
-                return;
-            }
-
-            var descriptor = Diagnostics.GetValueOrDefault(diagnosticName);
+            string methodFullName = $"{methodSymbol.ContainingType.GetFullName()}.{methodSymbol.GetDecoratedName()}";

```

Comment:
Aw, come on ^^
Do we have a rule about this? I typically use `var` for most things, except for built-in primitive types